src/main.ts

```diff
@@ -68,8 +68,12 @@ async function main() {
     port: parseInt(process.env.PORT || '0', 10),
   });
 
-  process.once('SIGINT', () => server.stop());
-  process.once('SIGTERM', () => server.stop());
+  process.once('SIGINT', () => {
+    server.stop().catch(err => console.error(err));
```

Comment:
May be able to replace this with:

```js
server.stop().catch(console.error);
```

- Not a blocker

Reply:
I got into a habit of avoiding passing references of `console.{log,error}` functions around as they sometimes error with "illegal invocation", I think it's an issue with how they generate the stack frames for the log entry (like new Error)